### PR TITLE
Completed Fetch and cleanup

### PR DESCRIPTION
**Pull request by cjhbh3**
Complete Fetch step of Week 3:
In script.js, added a fetch function using arrow implementation. Implemented fetch button on my portfolio that requests and outputs the text in /data

Other changes:
Fixed REAME.md - mistyped portfolio
Adjusted random greeting in script.js
Updated index.html with new formatting.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,21 +8,25 @@
...
+        Why did I start CS? My experience as a lead programmer in FTC Robotics during High School.</br>
       </p>
       <p>This is my dog, Juliette.</p>
       <img src="/images/Juliette.jpg" width="500" height="500">
       <p>Here is what I look like!</p>
       <img src="/images/SelfPic.JPG" with="500" height = "500">
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Fetching Text</p>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch Me that Text</button>
```
Comment (ptai7) [inline portfolio/src/main/webapp/index.html:26]: Is the goal of this "Fetch Me that Text" button to print "Hello CJ!" or add a new quote to the container of quotes that would be shown on the page when the "Give me a Quote" button is clicked?

The website currently prints "Hello CJ!" when the "Fetch me that Text" button is clicked due to the doGet from DataServlet.java.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,21 +8,25 @@
...
+      <div id="quote-container"></div>
+      <p>Click here for one of my favorite tv show quotes:</p>
+      <button onclick="addRandomGreeting()">Give me a Quote</button>
```
Comment (ptai7) [inline portfolio/src/main/webapp/index.html:29]: How is quote vs greeting is being used here?

Comment (cjhbh3) [reply portfolio/src/main/webapp/index.html:29]: The quote fetches the text that is located on the /data page, while the greeting is randomly generated quote of four potential. I can update the function names to reflect these better.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (cjhbh3) [reply portfolio/src/main/webapp/index.html:26]: As far as I understood from the tutorial, the goal of the fetch() function was to grab the text from the /data page and print it. In my website, it should print "Hello CJ!".

Comment (ptai7) [reply portfolio/src/main/webapp/index.html:26]: Noted, adding a comment on the purpose of the fetch according to the tutorial is useful.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ptai7) [reply portfolio/src/main/webapp/index.html:29]: Noted, please update the function names and/or add comments.

Comment (cjhbh3) [reply portfolio/src/main/webapp/index.html:29]: I will go ahead and do that in the next commit.

**Review by ptai7 — APPROVED** [review]

**Pull request merged**